src/control/cmd/daos_server/README.md

```diff
@@ -0,0 +1,180 @@
+# DAOS Server
+
+The DAOS Server can be run either as a foreground process to issue hardware
+provisioning commands or as a background process (binary name followed by the
+"start" subcommand) when it will listen for instructions from the [management
+tool](/src/control/cmd/dmg/README.md) and manage bring-up of the DAOS system.
+A DAOS Server instance will typically be run on every storage node to
+orchestrate the operation of the managed data plane processes.
+
+## Network and Storage Management
+
+The DAOS data plane utilises two forms of non-volatile storage,
+storage class memory (SCM) in the form of persistent memory modules
+and NVMe in the form of high-performance SSDs.
+
+The DAOS Server provides capability to provision and manage network
+devices and non-volatile storage including the allocation of
+resources to data plane instances.
+
+See
+[admin guide](https://daos-stack.github.io/admin/deployment/#hardware-provisioning)
+for details on hardware provisioning.
+
+## Configuration
+
+A server configuration file which is passed on invokation contains details of
+global and per-io-server parameters related to hardware devices and
+environments.
+
+### Configuration File
+
+A populated `daos_server` config file is required when starting
+`daos_server` process, it's location can be specified on the
+commandline (`-o` option) or default location
+(`<daos install dir>/install/etc/daos_server.yml`).
+
+Example config files can be found in the
+[examples folder](https://github.com/daos-stack/daos/tree/master/utils/config/examples).
+
+Config file parameters will be parsed and populated with defaults as
+documented in the
+[default daos server config](https://github.com/daos-stack/daos/tree/master/utils/config/daos_server.yml).
+
+Parameters passed to `daos_server` on the commandline as application
+options (excluding environment variables) take precedence over values
+specified in config file and for convenience, active parsed config
+values are written to the directory where the server config file was
+read from or `/tmp/` if that fails.
+
+It is strongly recommended to specify all parameters and environment
+for running DAOS servers in the
+[server config file](https://github.com/daos-stack/daos/tree/master/utils/config/daos_server.yml).
+
+!!! NOTES:
```

Comment:
Is this document intended to be viewed via github or will it be included in the admin guide? This markdown isn't valid for github.